### PR TITLE
Update security.rst

### PR DESCRIPTION
**Pull request by norbert-n**
missing use statement

### CODE REVIEW

**Comment by OskarStark** [comment]
LGTM 👍

File: book/security.rst [hunk 1]
```diff
@@ -185,6 +185,7 @@ example, if you use annotations, create something like this::
 
     use Sensio\Bundle\FrameworkExtraBundle\Configuration\Route;
     use Symfony\Component\HttpFoundation\Response;
+    use Symfony\Bundle\FrameworkBundle\Controller\Controller;
```
Comment (xabbuh) [inline book/security.rst]: Can you please sort the `use` statements alphabetically?

Comment (norbert-n) [inline book/security.rst]: done

**Comment by javiereguiluz** [comment]
It looks good ... but I don't know if doc maintainers will agree with it. The `// ...` at the beginning of the code is used in the Symfony Docs to refer to *"all the "use" that are needed but we don't show them here because they are very basic or they were showed in the previous examples"*.

**Comment by WouterJ** [comment]
Personally, I only apply `// ...` to the "shown before" standard, not the "very basic". For beginners, nothing is basic in Symfony. So a :+1: for me.

**Comment by xabbuh** [comment]
I absolutely agree with @WouterJ. This is the book which is probably one of the first parts newcomers are reading. And secondly this is the first controller example in this chapter. So :+1: from me too.

**Comment by xabbuh** [comment]
Thank you @norbert-n. I have merged your fix into the `2.7` branch (that's why it's shown as closed instead of merged) as the `2.3` branch is no longer maintained.

**Pull request closed without merging**